src/components/views/messages/MPollBody.tsx

```diff
@@ -380,7 +444,7 @@ export default class MPollBody extends React.Component<IBodyProps, IState> {
         const newEvents: MatrixEvent[] = this.state.voteRelations.getRelations()
             .filter(isPollResponse)
             .filter((mxEvent: MatrixEvent) =>
-                !this.seenEventIds.includes(mxEvent.getId()));
+                !this.seenEventIds.includes(mxEvent.getId()!));
```

Comment:
How do we know, that event Id is always set here?

Reply:
We don't know whether it will be set but it doesn't matter when we use it in a `[1,2,3].includes(null)`. `null` won't match anything

Another way to thin about it, an event without an ID won't be in our `seenEventIds` anyway.